src/components/PhoneNumber.tsx

```diff
@@ -1,27 +1,44 @@
 import {useState} from "react";
 import "./phoneNumber.scss";
+import PhoneInput from 'react-phone-input-2';
```

Comment:
react-phone-input-2 - there is no such module./

You need to add this npm dependancy in package.json to this pull request

Reply:
Dıd you fıxed thıs comment?

Reply:
"react-datepicker": "^4.8.0",
    "react-dom": "^18.2.0",
    "react-feather": "^2.0.10",
    "react-phone-input-2": "^2.15.1",
    "react-redux": "^8.0.4",
    "react-router-dom": "^6.4.2",
    "react-scripts": "5.0.1",